docs/rules/no-namespace-imports.md

```diff
@@ -0,0 +1,46 @@
+# Prevent namespace import declarations. (no-namespace-imports)
+
+Namespace imports grab all exported variables from a module as nested properties under a single variable name. This approach of importing everything, rather than only what is required from the module, leads to a polluted local scope from the unused components that were brought in under the module namespace.
```

Comment:
It does not polite the local scope, that's half the benefit of namespace imports. If we're going to sell this on anything, it should be that this style of import breaks tree shaking.